src/internal/operations/backup_integration_test.go

```diff
@@ -70,22 +75,43 @@ func prepNewBackupOp(
 		ms.Close(ctx)
 	}
 
-	sw := store.NewKopiaStore(ms)
+	bo := newBackupOp(t, ctx, kw, ms, acct, sel, bus, featureFlags, closer)
 
-	bo, err := NewBackupOperation(
-		ctx,
-		control.Options{},
-		kw,
-		sw,
-		acct,
-		sel,
-		bus)
+	return bo, acct, kw, ms, closer
+}
+
+// newBackupOp accepts the clients required to compose a backup operation, plus
+// any other metadata, and uses them to generate a new backup operation.  This
+// allows backup chains to utilize the same temp directory and configuration
+// details.
+//
+//revive:disable:context-as-argument
+func newBackupOp(
```

Comment:
`newBackupOp` --> `testBackupOp`

Reply:
👍 Like the other PR, I'll save the rename here for the next PR.